lib/api.js

```diff
@@ -68,6 +72,17 @@ function parseApiResponse(respData, callback) {
   callback(null, respData.result);
 }
 
+function pushOntoRetryQueue() {
+  retryQueue.push(arguments);
+
+  if (!retryHandle) {
+    retryHandle = setInterval(function() {
```

Comment:
Yeah, my thought was that it is essentially a no-op, so I kept the code simple.  If we want it, I can make it stop.  Think that would be better?